cmd/catalyst-node/catalyst-node.go

```diff
@@ -284,28 +296,26 @@ func main() {
 		return
 	}
 
-	if *version {
+	if cliFlags.Version {
 		fmt.Println("catalyst-node version: " + Version)
 		fmt.Printf("golang runtime version: %s %s\n", runtime.Compiler, runtime.Version())
 		fmt.Printf("architecture: %s\n", runtime.GOARCH)
 		fmt.Printf("operating system: %s\n", runtime.GOOS)
 		return
 	}
 
+	cliFlags.RedirectPrefixes = strings.Split(*prefixes, ",")
+	glog.Infof("found redirectPrefixes=%v", cliFlags.RedirectPrefixes)
```

Comment:
Maybe a bit less verbose?
```suggestion
	glog.V(6).Infof("found redirectPrefixes=%v", cliFlags.RedirectPrefixes)
```